Dynavity/Dynavity.xcodeproj/project.pbxproj

```diff
@@ -531,17 +535,20 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
+				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_ASSET_PATHS = "\"Dynavity/Preview Content\"";
+				DEVELOPMENT_TEAM = STWSL7H5JX;
```

Comment:
Not too sure if we should be committing this change since everyone else will face conflicts using this config.

Reply:
Hmm alright, will try to find a workaround for this, otherwise each time I will have to change the team name before the last commit since I'll need to change the DEVELOPMENT_TEAM to run it on my iPad